src/services/ethconversion/ethconversion.hooks.js

```diff
@@ -0,0 +1,52 @@
+import onlyInternal from '../../hooks/onlyInternal';
+import { getEthConversion } from './getEthConversionService';
+
+const getConversionRates = () => (context) => {
+  const { app, params } = context;
+
+  console.log(params);
+
+  // block internal calls
+  if (!params.provider && !params.internal) return context
```

Comment:
why do we need to block internal calls?

Reply:
Because the getEthConversion in getEthConversionService also does a find, and that results in recursion. So in case of internal calls we just return the context, unless you pass in `{ internal: true }`